<launch>
  <node name="luna_gui_gw" pkg="lunabotics" type="luna_gui_gw" />
  <node name="luna_gui_listener" pkg="lunabotics" type="luna_gui_listener" />
</launch>
